includes/cart/functions.php

```diff
@@ -558,6 +558,28 @@ function edd_get_cart_item_price_name( $item = array() ) {
 }
 
 /**
+ * Get cart item title
+ *
+ * @since 2.4.2
+ * @param int $item Cart item array
+ * @return string item title
+ */
+function edd_get_cart_item_title( $item = array() ) {
+	$item_title = get_the_title( $item['id'] );
+
+	if( empty( $item_title ) ) {
+		$item_title = $item['id'];
+	}
+
+	if ( edd_has_variable_prices( $item['id'] ) && edd_get_cart_item_price_id( $item ) !== false ) {
```

Comment:
Can we write that 2nd condition as `false !== edd_get_cart_item_price_id( $item )`

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions